tests/integrations/linuxdeploy/utils.py

```diff
@@ -0,0 +1,43 @@
+from random import randrange
+from pathlib import Path
+
+PATCH = {
+    'offset': 0x08,
+    'original': bytes.fromhex('414902'),
+    'patch': bytes.fromhex('000000')
+}
+
+
+def create_mock_appimage(appimage_path: Path, mock_appimage_kind: str = 'original'):
+    """
+    Create a mock AppImage for testing purposes.
+
+    Args:
+        appimage_path (Path): Path to the appimage to create.
+        mock_appimage_kind (str): The kind of mock appimage to create.
+            'original' creates an unpatched mock appimage.
+            'patched' creates a patched mock appimage.
+            'corrupt' creates a corrupted mock appimage.
+
+    Returns:
+        bytes_to_be_patched (bytes): The bytes to be patched of the created AppImage.
```

Comment:
These docs are great, but we use a slightly different format; see [this method](https://github.com/beeware/briefcase/blob/master/src/briefcase/commands/create.py#L75) for an example

Reply:
I've updated the documentation to conform to the project's documentation pattern.